sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/windowExpressions.scala

```diff
@@ -348,16 +348,12 @@ abstract class OffsetWindowFunction
 
   /**
    * (Foldable) expression that contains the number of rows between the current row and the row
-   * where the input expression is evaluated.
+   * where the input expression is evaluated. If `offset` is a positive integer, it means that
+   * the direction of the `offset` is from front to back. If it is a negative integer, the direction
+   * of the `offset` is from back to front.
```

Comment:
what if the offset is 0?

Reply:
`If it is zero, it means that the offset is ignored and use current row.`